### PR TITLE
Env vars for local paths, chunking and embedding are optional, misc clean-up for consistency

### PR DESCRIPTION
**Pull request by Paul-Cornell**
- `input_path="some-random-dir"` -> `input_path=os.getenv("LOCAL_FILE_INPUT_DIR")`
- `download_dir="some-random-dir"` -> `download_dir=os.getenv("LOCAL_FILE_DOWNLOAD_DIR")`
- `output_dir="some-random-dir"` -> `output_dir=os.getenv("LOCAL_FILE_OUTPUT_DIR")`
- `--input-path "some-random-dir"` -> `--input-path $LOCAL_FILE_INPUT_DIR`
- `--download-dir "some-random-dir"` -> `--download-dir $LOCAL_FILE_DOWNLOAD_DIR`
- `--output-dir "some-random-dir"` -> `--output-dir $LOCAL_FILE_OUTPUT_DIR`
- Spell out embedding provider (i.e. `$EMBEDDING_PROVIDER` -> `langchain-huggingface`).
- Added comments that chunking and embedding are optional.
- Consistent whitespace treatment in CLI commands, for better readability.
- Removed extraneous quotation marks in CLI commands, for consistency.

### CODE REVIEW

**Review by MKhalusova — APPROVED** [review]
LGTM!

File: snippets/destination_connectors/elasticsearch.sh.mdx [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 unstructured-ingest \
   local \
-    --input-path example-docs/book-war-and-peace-1225p.txt \
-    --output-dir local-output-to-elasticsearch \
+    --input-path $LOCAL_FILE_INPUT_DIR \
+    --output-dir $LOCAL_FILE_OUTPUT_DIR \
     --strategy fast \
     --chunk-elements \
-    --embedding-provider "$EMBEDDING_PROVIDER" \
+    --embedding-provider $EMBEDDING_PROVIDER \
```
[the remark this answers is not in the document]
Comment (Paul-Cornell) [reply snippets/destination_connectors/elasticsearch.sh.mdx:12]: Good catch--thanks!

File: snippets/destination_connectors/gcs.sh.mdx [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+    --strategy fast \
+    --chunk-elements \
+    --embedding-provider langchain-huggingface \
+    --num-processes 2 \
+    --verbose \
   gcs \
-  --service-account-key "$SERVICE_ACCOUNT_KEY" \
-  --remote-url "<your destination path here, ie 'gcs://unstructured/war-and-peace-output'>"
+    --service-account-key $SERVICE_ACCOUNT_KEY \
+    --remote-url "gcs://unstructured/MY-output"
```
[the remark this answers is not in the document]
Comment (Paul-Cornell) [reply snippets/destination_connectors/gcs.sh.mdx:17]: 🤷  -- thanks for the catch!

Comment (Paul-Cornell) [inline snippets/destination_connectors/gcs.sh.mdx:17]:
```suggestion
    --remote-url "gcs://unstructured/my-output"
```

File: snippets/destination_connectors/elasticsearch.sh.mdx (hunk 1 above)
Comment (Paul-Cornell) [inline snippets/destination_connectors/elasticsearch.sh.mdx:12]:
```suggestion
    --embedding-provider langchain-huggingface \
```

**Pull request merged**